modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/data_renderers/index.js

```diff
@@ -47,7 +47,7 @@ const dataRenderers = {
 
 export const inputRenderers = {
 	checkbox: InputCheckboxRenderer,
-	dateTime: InputDateTimeRenderer,
+	dateTimeInput: InputDateTimeRenderer,
```

Comment:
@oliv-yu This looks good, renderer IDs need to be unique. But, I'm a bit worried, is `InputDateTimeRenderer` used anywhere? Do we use date-time picker input for inline editing anywhere? If yes, this will cause regressions; those usages would need to be updated to have `dateTimeInput` as id. Looking at when we use `TableInlineAddingRow`, it seems this [depends on `inlineEditSettings`](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/table/Table.js#L135-L140). The only place I see this used is here in commerce dev environment, [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/commerce/commerce-frontend-js/dev/components/DatasetDisplay.js#L829) and [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/commerce/commerce-frontend-js/dev/components/DatasetDisplay.js#L838). We should probably update there. Are there any other usages?

Reply:
@markocikos Oo, I'm actually not sure if inline editing is being used, just because something like "Created Date" doesn't seem like it would want to be manually updated.

I added the change to the document (it looked like there were three places the `dateTime` was being used?), but just in case, would like to mention @ivicac who last edited this commerce DatesetDisplay.js. Ivica, would you mind reviewing this latest commit? Feel free to let us know if there are other places that need to be updated for the new `dateTimeInput` id.

Thanks!!

Reply:
@FabioDiegoMastrorilli could you please review the commit @oliv-yu mentioned?
thx

Reply:
@oliv-yu I would try a different approach since the purpose of that object is to contain all input renderers. Adding `input` looks redundant.

Keep the property as it is and remove:
```
export default {
	...dataRenderers,
	...inputRenderers,
};
```
I think you can export directly `dataRenderers` and update the imports in 
`modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/utils/dataRenderers.js` and 
`modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/index.js`

this way you'll make the dataset safer and delete some unsafe code (that I wrote :) )

Reply:
Thanks for the approach, Fabio!

Updated the branch, let us know if this works!

Reply:
It is exactly what I thought :D 
If it ok after testing, go with it; you have my blessing :)